website/src/components/WindowedPagination/WindowedPagination.js

```diff
@@ -19,7 +19,8 @@ export const WindowedPagination = React.memo(({
     justifyContent: space-between;
     width: 100%;
     margin: 1em 0;
-  '>
+  '
+  >
```

Comment:
I think this is a mistake, no?

Reply:
Yeah, seems to be, my bad. Done in https://github.com/1Hive/apiary/pull/181/commits/5b75dfab2dea2611ec54dee8bcee7008a4dd64aa.